behat.yml.dist

```diff
@@ -400,6 +400,7 @@ default:
     extensions:
         Lakion\Behat\MinkDebugExtension:
             directory: etc/build
+            clean_start: false
```

Comment:
I've smuggled a tiny change here to make logs of failing scenarios available (as `MinkDebugExtension.clean_start=true` by default and it cleared the `etc/build/` directory).